Assets/Scripts/IUnityService.cs

```diff
@@ -0,0 +1,26 @@
+using UnityEngine;
+
+public interface IUnityService
+{
+    float GetDeltaTime();
```

Comment:
Move it to another interface to follow Interface Segregation Principle. It might be called `ITimeService` for example.
```suggestion
```